src/main/java/gui/MainFrame.java

```diff
@@ -26,26 +30,30 @@
 	private JButton checkQualityButton;
 	private JComboBox<Object> rulesDropDown;
 	private ArrayList<CodeQualityRule> rulesList;
+	private QualityIndicator qualityIndicators;
 
 	/**
 	 * The mainframe constructor receives an excel table, generated from the given
-	 * excel file, as well as a list containing all the rules, stored in the main
+	 * excel file, as well as a list containing all the rules and quality indicators, stored in the main
 	 * controller
 	 * 
 	 * @param excelTable
 	 * @param rulesList
+	 * @param indicatorsQuality

```

Comment:
Adiciona uma descrição tbm sobre o teu novo parametro. Vamos ter que fazer pro todos eventualmente. Tipo
```suggestion
	 * @param indicatorsQuality A QualityIndicator object that potatoes
```

Reply:
done